packages/xo-server/src/xapi/index.js

```diff
@@ -264,7 +264,7 @@ export default class Xapi extends XapiBase {
         pool.set_name_description(nameDescription),
       nameLabel !== undefined && pool.set_name_label(nameLabel),
       autoPoweron != null &&
-        pool.update_other_config('autoPoweron', autoPoweron ? 'true' : null),
+        pool.update_other_config('auto_poweron', autoPoweron ? 'true' : null),
```

Comment:
This should not be here, maybe an issue in your rebase.

Reply:
I will point this branch to nayraud branch

Reply:
Still here.

Reply:
:heavy_check_mark:  Fixed

Reply:
Nope, please read your PR changelog.